### ISSUE TITLE
Add the ability to transfer images to more than one helioviewer server

### ISSUE THREAD

**Issue by wafels** [Helioviewer-Project/jp2gen#36]
There are currently two helioviewer servers at GSFC. Both need to be receiving new images from the JP2 generation computer. Therefore the JP2Gen generation computer needs to be able to transfer to multiple machines. Currently JP2Gen can only move data to a single machine.

A potential solution is to change the values in the `remote` structure (https://github.com/Helioviewer-Project/jp2gen/blob/master/idl/local/EXAMPLE_hv_writtenby.pro#L52) into arrays, and then have the transfer script `hv_transfer_jp2.pro` (https://github.com/Helioviewer-Project/jp2gen/blob/master/idl/gen/hv_jp2_transfer.pro#L73) step through all the remote locations indicated in the modified `hv_writtenby.pro` file. I think a simple `for...endfor` loop to index the remote location data should be sufficient.

**Comment by dzarro** [comment]
This looks like something Jonathan can tackle.

**Comment by jonltsang3** [comment]
I'm on it now

**Comment by dgarciabriseno** [comment]
I don't think this was ever completed, and it didn't meet our deadline back in May anyway. I've pretty much worked around this by adding helioviewer.org itself as a [source for certain images](https://github.com/Helioviewer-Project/api/tree/master/install/helioviewer/hvpull/servers). Mirrors can use the hv_* server definitions to pull data from helioviewer. This is how I get STEREO and SOHO data onto the beta server.
With that, we can probably close this unless you can think of a good reason for still needing it.
